src/en/proposals/rainfall-vampires.md

```diff
@@ -0,0 +1,112 @@
+The injuries you inflict are obvious, you leave evidence of your actions and your mouth will need to be uncovered to feed.
+
+You have no abilities that increase your resistance to damage or conceal your identity, and minimal stealth abilities.
+
+The Vampire's core powerset is focused on evading/escaping capture, disabling a single target to feed upon and limited revival from death.  
+To a Vampire, blood is everything - They require it for sustenance, purchasing new abilities and using existing abilities.  
+As such a vampire must balance how often they use their abilities - too often and they will be unable to make new purchases.
+
+Should they run out of blood, they will be easy prey for the crew.
+
+Directly countering the Vampire is the Chaplain upon whom Vampire abilities are useless, and their Bible which will greatly injure any Vampire it hits.
+
+## Innate Traits
+
+- Vampires appear - for the most part - to be normal crew, only distinguishable by their fangs being visible when extended.
+- They cannot eat solid foods, only liquid and pills
+- Immune to freezing
+- Immune to barotrauma
+- Cannot rot
+- Are harmed by interactions with anything Holy (Bibles, Holy Water)
+- Cannot leave the station, as they are harmed by starlight
+
+## Blood Drinking
+Feeding on the living requires they be restrained or otherwise incapacitated - lest they attempt to escape.  
+You are limited by your stomach capacity, which means you can only ingest about 30-50u of blood at a time - roughly a safe amount to take from a healthy victim.  
+Fully draining a victim will take ~120 seconds - including breaks to digest your meal - and net you 300 units of blood.
+
+## Heirloom
+The heirloom is the vampire's means of purchasing new abilities. Its innately bound to the vampire and cannot be used by others.
+Should the vampire lose it, they may summon it to their hand.
+
+## Abilities
+
+Vampires get access to a mix of active and passive abilities:
+
+### Actives
+
+- Toggle Fangs  
+Extend or retract your fangs, fangs are visible while extended. Must be extended to drink blood  
+
+- Glare  
+Stun your target for 10 seconds, blocked by eye protection  
+60 Second cooldown
+
+- Hypnotise  
+Stare into your targets eyes for 5 seconds, after which they are forced to sleep for 1 minute 
+5 Minute cooldown
+
+- Screech  
+Emit a loud piercing screech that knocks everyone nearby to the ground for 3 seconds.  
+Will also damage all fragile items or windows in range.  
+10 Blood activation cost  
+60 Second cooldown
+
+- Blood Steal  
+Steal blood from all humanoids nearby, quite obvious to everyone nearby.  
+20 Blood activation cost  
+60 Second cooldown
+
+- Polymorph: Bat  
+Transform yourself into a Bat.  
+Fast, able to fly. No other abilities are usable in this form.  
+30 Second cooldown
+
+- Polymorph: Mouse  
+Transform yourself into a Mouse.  
+Small, able to move under doors. No other abilities are usable in this form.  
+30 Second cooldown
+
```

Comment:
You shouldn't rely upon a bandaid for core gameplay, mouses being able to go through doors is subject to removal at any time.

Reply:
Is it likely to be removed before we have a ventcrawling implementation?

Considering the other polymorph (bat)'s only advantage is it 'flying', i would consider a mouse being able to move under tables to be roughly equal in utility.